client/contivModelClient.go

```diff
@@ -80,7 +136,16 @@ type Link struct {
 
 func httpGet(url string, jdata interface{}) error {
 
-	r, err := http.Get(url)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return err
+	}
+
+	if err := addAuthTokenHeader(req); err != nil {
```

Comment:
Are we enforcing that all the requests should go through proxy? What if the user wants to bypass proxy and make a call to netmaster directly?